to_production.txt

```diff
@@ -1,8 +1,11 @@
 # New
+ofl/afacadflux # https://github.com/google/fonts/pull/7851
 ofl/eduauvicwantdots # https://github.com/google/fonts/pull/8020
 ofl/eduauvicwantguides # https://github.com/google/fonts/pull/8021
+ofl/sixtyfourconvergence # https://github.com/google/fonts/pull/7919
```

Comment:
COLR family that needs to be on the WS editors denylist.  I reached out to Evan about this yesterday